app/js/fullscreen.ts

```diff
@@ -0,0 +1,13 @@
+function enterFullscreen() {
+  if (!document.fullscreen) {
+    document.documentElement.requestFullscreen();
+    document.onfullscreenchange = function () {
+      document.documentElement.removeEventListener(
+        "mousemove",
+        enterFullscreen
+      );
+    };
+  }
+}
+
+document.documentElement.addEventListener("mousemove", enterFullscreen);
```

Comment:
nothing is happening for us on mouse move but the window goes to fullscreen on our first click. If it comes out of fullscreen it cannot be re-entered.